client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -45,6 +45,42 @@ const generateSearchLink = (title: string, block: string) => {
   return search;
 };
 
+interface CheckbockProps {
+  name: string;
+  i18nkey: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: boolean;
+  href: string;
+  t: (text: string) => string;
+}
+
+function CheckboxHelpModal({
+  name,
+  i18nkey,
+  onChange,
+  value,
+  href,
+  t
+}: CheckbockProps) {
+  return (
+    <div style={{ display: 'flex', flexDirection: 'row' }}>
+      <input
+        name={name}
+        type='checkbox'
+        onChange={onChange}
+        checked={value}
+        aria-label={t(i18nkey)}
```

Comment:
```suggestion
        aria-label={t(i18nkey)}
        aria-required='true'
```

You can't submit without checking these, so we should have `aria-required` on them.